public/static/locales/en/profile.json

```diff
@@ -147,6 +147,12 @@
         "setAsPrimaryAddress": "Set as Primary Address",
         "setAsBillingAddress": "Set as Billing Address"
       },
+      "deleteAddress": "Delete Address",
+      "delete": "Delete",
+      "addressConfirmationMessage": "Are you sure you want to set this address as your {addressType, select, mailing {billing} other {{addressType}}} address? {isAddressSet, select, true {This will replace your current {addressType, select, mailing {billing} other {{addressType}}} address.} other {}}",
+      "deleteAddressConfirmationMessage": "Are you sure you want to delete this address? If you want to use it again, please add it as a new address.",
+      "confirm": "Confirm",
+      "address2": "Address 2(optional)",
```

Comment:
Do some of the keys that are currently outside need to be moved/grouped?

For example, I see `delete`, and `address2` keys